code/script.js

```diff
@@ -30,18 +35,117 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+function toggleVisible(className) {
+  document.querySelector(`.${className}`).classList.toggle("invisible");
+}
+
+
+// 1. Starts here question 1
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Welcome to the Time Bank, What's your name?`, 'bot')
 }
+setTimeout(greeting, 500);
 
-// Set up your eventlisteners here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+// 2. Read the input, show the input, show question 2, change the options
+const handleNameInput = (event) => {
+  event.preventDefault();
+  //read the input 
+  const nameInput = document.getElementById("name-input");
+  const name = nameInput.value;
+  showMessage(name, 'user');  //print the name in screen 
+  nameInput.value = ''
+
+  setTimeout(() => showTimeOptions(name), 1000);
+}
+
+const showTimeOptions = (name) => {
+  showMessage(`Hello ${name}! I'm happy you have found your way to the Time Bank, welcome! How much time do you need? 1 hour, 1 day or 1 week?`, 'bot');
+  toggleVisible("question-name");
+  toggleVisible("question-time");
```

Comment:
Looks as a very creative solution! I would love to try my bot this way, thank you for inspiration!